config/v1.5/aws-k8s-cni.yaml

```diff
@@ -98,6 +98,10 @@ spec:
           env:
             - name: AWS_VPC_K8S_CNI_LOGLEVEL
               value: DEBUG
+            - name: AWS_VPC_K8S_CNI_VETHPREFIX
```

Comment:
IMO, I would like to move these variables into a configMap so that they persisted upon upgrades. May be a separate PR and add //To do here  ?

Reply:
ack. good idea.